iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/BranchObservabilityImpl.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.impl.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.Branch;
+import com.powsybl.iidm.network.extensions.BranchObservability;
+import org.jgrapht.alg.util.Pair;
+
+import java.util.EnumMap;
+import java.util.Map;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public class BranchObservabilityImpl<T extends Branch<T>> extends AbstractExtension<T>
+        implements BranchObservability<T> {
+
+    private boolean observable;
+
+    private final Map<Branch.Side, Pair<Double, Boolean>> standardDeviationP = new EnumMap<>(Branch.Side.class);
```

Comment:
Same comment as in the injection part: having plain fields seems simpler / more memory efficient.